recipes/mcstas-readout-master/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "mcstas-readout-master" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/g5t/mcstas-readout-master/archive/v{{ version }}.tar.gz
+  sha256: 64dea61e66f2fa71efa4fb32caeb81f6640028bc26600510aceb861352e81572
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mcstas-readout-master') }}
+  ignore_run_exports_from:
+    - python
+
+requirements:
+  build:
+    - python
```

Comment:
Just to check, python is only a build dependency and you understand the distinction between `build`/`host` for Python? Not a problem just unusual 😄 

(if my question isn't clear ask and I'll elabarate)

Reply:
Python is only used through CMake to configure a header file with version and build information.

It was suggested to me that it should be a `host` dependency -- but that caused multiple identical builds, needlessly using build resources. I can move it back if you think the version of Python that configured a header file could impact the resulting shared library or binaries.
(To answer your question more directly: Do I understand the distinction between `build`/`host` for Python? Mostly :shrug: )

Reply:
Perfect you're doing the right thing! 😄